exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/using_a_script_languages_container.ipynb

```diff
@@ -0,0 +1,704 @@
+{
```

Comment:
<p>The sytax of the activation statement is ... "The alias is used when defining an UDF, the URL only for the language activation." The meaning of this sentence is not clear. Do we actually need it? "&nbsp;For simplicity the <code>SlctManager</code> provides easy access to the alias" This sentence is repeated below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>